client/components/forms/form-setting-explanation/index.tsx

```diff
@@ -0,0 +1,18 @@
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+
+import './style.scss';
+
+const FormSettingExplanation = forwardRef<
+	HTMLParagraphElement,
+	{ children: React.ReactNode; className?: string; noValidate?: boolean; isIndented?: boolean }
```

Comment:
Worth supporting the rest props properly?

```suggestion
	{ noValidate?: boolean; isIndented?: boolean } & React.ComponentProps< 'p' >
```

Reply:
I was about to reply but Jarda [got there first](https://github.com/Automattic/wp-calypso/pull/98702#issuecomment-2606550637) — I think I'll remove the rest props altogether since they don't seem to be needed

Reply:
One more super-tiny improvement you can do is to remove the `className="feed-settings__excerpt-explanation"` prop from one of the usages because the referred CSS class doesn't exist. I randomly noticed that when skimming through the component usages 🙂

Reply:
> I think I'll remove the rest props altogether since they don't seem to be needed

No strong opinion for this particular case, but why though? It's a standard open/closed pattern, and very straightforward in this case. Just because nobody uses it now, doesn't mean they should have to modify the component when they want `id` or `data-testid` at some point.